python/test_dcr_template.py

```diff
@@ -358,35 +382,6 @@ def test_persist_dcr_model_roundtrip(self):
         for key in param_ref.keys():
```

Comment:
This seems a rather weak test in that you are testing that the keys are present but not their values. Can you do better? Would it make sense to exercise the unpersisted model?